### PR TITLE
Destroy branch

### PR DESCRIPTION
**Pull request by jzoon**
"As an agent
When in a simulation
I am able to destroy stuff if I think 
this is benificial for me.
To make room for construction or to 
optimize a certain indicator."

### CODE REVIEW

**Comment by MichielvdBerg** [comment]
Approved

**Comment by tvleest** [comment]
approved.

**Comment by spiderbiggen** [comment]
Approved

File: TU-Agent/tygron.mod2g [hunk 1]
```diff
@@ -11,7 +11,7 @@ order=linear.
 module tygron {
 	
 	% Call the module demolish to demolish EWI
```
Comment (daanwerf) [inline TU-Agent/tygron.mod2g]: This comment is no longer valid, is it?

**Comment by daanwerf** [comment]
In another part of this weeks sprint the "if true" in the mod2g will have to be replaced by a goal representing the need to demolish something. Other than that, approved.

**Comment by jzoon** [comment]
I will change the comment and then merge this branch

**Pull request merged**